tests/foreman/ui/test_host.py

```diff
@@ -257,6 +257,34 @@ def test_positive_read_from_details_page(session, module_host_template):
         assert 'Admin User' in values['properties']['properties_table']['Owner']
 
 
+def test_read_host_with_ics_domain(
+    session, module_host_template, module_location, module_org, target_sat
+):
+    """Create new Host with ics domain name and verify that it can be read
+
+    :id: 54e3db92-16c2-412b-bf68-44d479c5987b
+
+    :expectedresults: Host ending with ics domain name can be accessed through Host UI
+    """
```

Comment:
```suggestion
    :Verifies: SAT-26202
    """
```

Reply:
Thanks, I keep forgetting how to do this lol